### PR TITLE
Bump karalabe/usb dependency to latest release

### PR DESCRIPTION
**Pull request by mdehoog**
Currently quorum doesn't compile on Mac OSX 12.5, due to the following error:

```
# github.com/karalabe/usb
In file included from ../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libs.go:50:
../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libusb/libusb/os/darwin_usb.c:253:39: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:123:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
In file included from ../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libs.go:50:
../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libusb/libusb/os/darwin_usb.c:390:26: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:123:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
In file included from ../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libs.go:50:
../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libusb/libusb/os/darwin_usb.c:441:60: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:123:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
In file included from ../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/libs.go:51:
../../go/pkg/mod/github.com/karalabe/usb@v0.0.0-20190919080040-51dc0efba356/hidapi/mac/hid.c:693:34: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:123:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
```

This was fixed in https://github.com/karalabe/usb/pull/26, and released in `v0.0.2`.

### CODE REVIEW

**Comment by antonydenyer** [comment]
Thanks @mdehoog is there any chance you could cherry-pick the commit from geth upstream? It makes merging upstream changes easier.

**Comment by mdehoog** [comment]
Hey @antonydenyer, thanks for the suggestion. The relevant commit is https://github.com/ethereum/go-ethereum/commit/fb3a6528cfa49f623570575c4fe9e8a716cfcdf7.

Cherry picking is not possible with the current version of geth in this repo, as there has been quite a few `go.mod`/`go.sum` changes in between these commits and therefore the cherry pick has conflicts.

Also happy to wait for quorum to upgrade to geth `v1.10.16` to receive this change.

**Comment by antonydenyer** [comment]
Okay fair enough `v1.10.16` might be a while! We can merge this in after https://github.com/ConsenSys/quorum/pull/1469 has been released. 

(changing to draft until  https://github.com/ConsenSys/quorum/pull/1469)

**Review by antonydenyer — APPROVED** [review]

**Review by baptiste-b-pegasys — APPROVED** [review]

**Comment by mdehoog** [comment]
rebased on master after #1469 was merged

**Pull request merged**
